rites/roman1969/src/utils/dates.ts

```diff
@@ -293,8 +293,9 @@ export class Dates {
     year = this.#isLiturgicalYear ? this.#year - 1 : this.#year
   ): Date | null => {
     const id = `${year}_${dayOfOctave}`;
-    if (this.#weekdayWithinOctaveOfChristmas[id] !== undefined) {
-      return this.#weekdayWithinOctaveOfChristmas[id];
+    const existing = this.#weekdayWithinOctaveOfChristmas[id];
```

Comment:
I did not change this file. Is this really necessary? :thinking:

Reply:
yes, we can all tell from the commits you didn't change it... it's not _necessary_ but it does help understand the point, given that the same reference is made multiple times, and a few of these lines are bad patterns (returning assignments) that reference this, and i didn't want to make logical changes about it....